molecule/default/requirements.yml

```
@@ -1,8 +1,6 @@
 ---
 - src: https://github.com/cisagov/ansible-role-pip
   name: pip
```

Coupled with my previous suggestion.

```suggestion
  name: pip
- src: https://github.com/cisagov/ansible-role-python
  name: python
```

---

Please see commit d147f24.